src/libs/actions/IOU.ts

```diff
@@ -321,15 +321,35 @@ function getReportPreviewAction(chatReportID: string, iouReportID: string): Onyx
  * @param reportID to attach the transaction to
  * @param policy
  * @param isFromGlobalCreate
+ * @param shouldKeepExistingData
  * @param iouRequestType one of manual/scan/distance
- * @param skipConfirmation if true, skip confirmation step
  */
-function initMoneyRequest(reportID: string, policy: OnyxEntry<OnyxTypes.Policy>, isFromGlobalCreate: boolean, iouRequestType: IOURequestType = CONST.IOU.REQUEST_TYPE.MANUAL) {
+function initMoneyRequest(
+    reportID: string,
+    policy: OnyxEntry<OnyxTypes.Policy>,
+    isFromGlobalCreate: boolean,
+    shouldKeepExistingData: boolean,
```

Comment:
Let's not give the user the option to control this. It will create confusion. Instead we should determine this in the function itself